repos/spack_repo/builtin/packages/py_bottleneck/package.py

```diff
@@ -29,4 +31,14 @@ class PyBottleneck(PythonPackage):
     depends_on("py-versioneer", when="@1.3.3:", type="build")
     depends_on("py-numpy", type=("build", "run"))
     # https://github.com/pydata/bottleneck/issues/453
-    depends_on("py-numpy@:1", when="@:1.3", type=("build", "run"))
+    depends_on("py-numpy@2:", type=("build", "run"), when="@1.5:")
+    depends_on("py-numpy@:1", when="@:1.3.8", type=("build", "run"))
+
+    def url_for_version(self, version):
+
+        url = "https://files.pythonhosted.org/packages/source/b/{0}/{0}-{1}.tar.gz"
```

Comment:
```suggestion
        url = "https://files.pythonhosted.org/packages/source/b/Bottleneck/{0}-{1}.tar.gz"
```
Doesn't change package name, only tarball name